### DISCUSSION TITLE
Can the --user switch be given with the --burn command?

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#40] [General]
Instead of specifying the user with the --customize command, can it be given in the --burn?  It isn't listed on https://github.com/gitbls/sdm/wiki/Burn-Scripts.

**Comment by gitbls** [comment]
At the current time, no. You can implement your own user management on the burn command with a burn script, but that's kind of a lot of work. That's what sdm-gburn does, which might actually come close to doing what you want, modulo your other item about password proompting.

I'll look into adding `--user` to the burn command.

**Comment by gitbls** [comment]
V7.2 enables `--user` on the burn command.

**Reply by dshaw619** [reply]
I've tested this successfully with the new v7.2.  Thanks.
